### DISCUSSION TITLE
Can I use any other component instead of rehype-katex for rendering latex?

### DISCUSSION THREAD

**Discussion by samipism** [remarkjs/remark#690] [Q&A] [answered]
Before the latest release, there were renderers instead of components and thus I just mapped math node into my custom Latex component. But, after the release, it didn't work. In many places, rehype-katex is used, but I can't use that because I have made my custom latex for supporting mhchem extension also. rehype-katex doesn't have mhchem extension. Please help.

**Comment by wooorm** [comment, marked as answer]
- `rehype-katex` has options that are passed through to KaTeX
- See the tests for a [component](https://github.com/remarkjs/react-markdown/blob/a2b611babce101fd1e7225f0720819c020c40d94/test/react-markdown.test.js#L819).

**Reply by samipism** [reply]
Thank you. The link for the test worked as expected.
